demosplan/DemosPlanCoreBundle/Application/FrontController.php

```diff
@@ -76,21 +58,41 @@ public static function bootstrap(): void
      */
     public static function console(string $activeProject, bool $deprecatedFrontcontroller = false): void
     {
+        if (!in_array(PHP_SAPI, ['cli', 'phpdbg', 'embed'], true)) {
+            echo 'Warning: The console should be invoked via the CLI version of PHP, not the '.PHP_SAPI.' SAPI'.PHP_EOL;
+        }
+
         set_time_limit(0);
 
+        require DemosPlanPath::getRootPath('vendor/autoload.php');
+
+        if (!class_exists(ConsoleApplication::class) || !class_exists(Dotenv::class)) {
+            throw new LogicException('You need to add "symfony/framework-bundle" and "symfony/dotenv" as Composer dependencies.');
+        }
```

Comment:
We technically don't need this as this FrontController should not be available outside of a demosplan-core with all dependencies installed. However, we should keep it to make diffing against the Symfony baseline easier.

Reply:
this is exactly my intention to keep it here